scripts/drop-oracle-tables.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
+export DYLD_LIBRARY_PATH=/opt/oracle/instantclient/
+DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+/opt/oracle/instantclient/sqlplus system/oracle@192.168.99.100:49161/xe < "$DIR/drop-tables.sql"
```

Comment:
This is hardcoding the IP address for Oracle. On my system, Oracle server is not running at the URL